### DISCUSSION TITLE
new toolbar spacing feels like a downgrade

### DISCUSSION THREAD

**Discussion by tovc** [ppy/osu#26297] [Ideas] [answered]
Previously, it was possible to simply move your mouse to the top of the screen and move left/right to toolbar buttons. With the new toolbar redesign, this is no longer possible and it feels awful

currently:

https://github.com/ppy/osu/assets/49025619/70292c10-2615-43f8-9e33-5d3a37e19848

previously:

https://github.com/ppy/osu/assets/49025619/d4f1b97a-c4e5-4f26-8b2c-96de892f928d

I don't take issue with the rounded redesign, but this feels like a net negative change

**Comment by bdach** [comment, marked as answer]
duplicate of https://github.com/ppy/osu/discussions/26198

**Reply by tovc** [reply]
ah, thought this was changed in today's update, so I didn't think to look back further
